core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -28,6 +28,7 @@ import java.util.concurrent.locks.ReentrantLock
 import com.yammer.metrics.core.Gauge
 import kafka.api.{ApiVersion, KAFKA_0_10_1_IV0, KAFKA_2_1_IV0, KAFKA_2_1_IV1, KAFKA_2_3_IV0}
 import kafka.common.OffsetAndMetadata
+import kafka.internals.generated.{GroupMetadataKey => GenGroupMetadataKey, GroupMetadataValue, OffsetCommitKey, OffsetCommitValue}
```

Comment:
small nit: For the requests/responses, we have added `Data` as a suffix to avoid collisions with internal classes. Perhaps, we could use `GroupMetadataKeyData` instead of `GenGroupMetadataKey` here.